ext/RMagick/rmdraw.c

```diff
@@ -945,7 +945,7 @@ VALUE Draw_annotate(
     unsigned long width, height;
     long x, y;
     AffineMatrix keep;
-    char geometry_str[50];
+    char geometry_str[100];
```

Comment:
@Watson1978 can you explain why `100` is the right number here? Why not `75` or `150`?

Reply:
https://github.com/rmagick/rmagick/blob/b4775c92d02bfb550cd941f093869aefcd19dc80/ext/RMagick/rmdraw.c#L987

This line can consume a lot of buffers in the method.

Choose a value that `NUM2ULONG` / `NUM2LONG` can convert without raising an exception.

https://github.com/rmagick/rmagick/blob/b4775c92d02bfb550cd941f093869aefcd19dc80/ext/RMagick/rmdraw.c#L974-L977

```
irb(main):015:0> a = (2**63).to_s
=> "9223372036854775808"
irb(main):016:0> b = (2**62).to_s
=> "4611686018427387904"
irb(main):017:0> "#{a}x#{a}+#{b}+#{b}".length
=> 79
```

Maybe `100` is enough.